model/patch_lifecycle.go

```diff
@@ -446,8 +446,13 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 		if err = tasksToInsert.InsertUnordered(sessCtx); err != nil {
 			return nil, errors.Wrapf(err, "error inserting tasks for version '%s'", patchVersion.Id)
 		}
+		if p.IsParent() {
```

Comment:
Is there a way to add a check(s) to this conditional to increase our chances of this not being done if there are no new aliases being added?

Reply:
Updated `FinalizePatch` to accept a list of the child patch IDs created by `ProcessTriggerAlias`; this list can be used to check if the document should be updated.